src/morse_code.rb

```diff
@@ -0,0 +1,60 @@
+@morse_code = {
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214766685-e3d29eec-ab54-4d98-b3a6-482d7ce8c1ae.png)

- [ ] Your readme file is not descriptive because you didn't any information in your readme file and it is looking like a template :fearful: Kindly add Title, description, Tech Stack, Key Features, Future Features, Getting Started Section, Authors Section, MIT link :+1: